test_scripts/TimeBeforeShutdown/003_SDL_saves_all_logs.lua

```diff
@@ -0,0 +1,31 @@
+----------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0117-configurable-time-before-shutdown.md
+--
+-- Description: Check SDL is able to save all logs during shut down sequence
+-- if the value of 'MaxTimeBeforeShutdown' parameter is big enough and 'FlushLogMessagesBeforeShutdown' = true
+--
+-- In case:
+-- 1. In SDL .ini 'FlushLogMessagesBeforeShutdown' = true and 'MaxTimeBeforeShutdown' = 60
+-- 2. HMI sends 'BC.OnExitAllApplications(IGNITION_OFF)' notification to SDL
+-- SDL does:
```

Comment:
is it possible that sdl takes more than 60 seconds to flush all logs, so it stops at 60 seconds and there are still more logs not written?

Reply:
ATF approach in https://github.com/LuxoftSDL/sdl_atf/pull/8 allows to have quite stable write speed. The value is defined in  https://github.com/LuxoftSDL/sdl_atf/blob/6e61fb28bec35b228ed9c01eb1170be097e95b27/atf_parallels/loop.sh#L18
If this value is decreased SDL would require more time to flush all logs and vise versa: if the value is increased SDL would require less time.

Current values: write speed in ATF and timeouts in scripts was found during experiments.

Reply:
@dboltovskyi got it. Thank you.